src/app/components/tech-blog/tech-blog.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import blogs from '../../../assets/json/blogs.json';
+
+@Component({
+  selector: 'app-tech-blog',
+  templateUrl: './tech-blog.component.html',
+  styleUrls: ['./tech-blog.component.css']
+})
+export class TechBlogComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+  
+  blogs: any = blogs;
```

Comment:
shall we place the instance variable  before the constructor, `in between line number 9 and 11`